app/controllers/bridges/lom/tasks_controller.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+module Bridges
+  module Lom
+    # rubocop:disable Metrics/ClassLength
+    class TasksController < ActionController::API
+      OML_SCHEMA_PATH = Rails.root.join('vendor/assets/schemas/lom_1484.12.3-2020/lom.xsd')
+
+      def show
+        builder = Nokogiri::XML::Builder.new(encoding: 'UTF-8') { |xml| sample_oml(xml) }
+
+        if params[:validate].present?
+          schema = Nokogiri::XML::Schema(File.read(OML_SCHEMA_PATH))
+          errors = schema.validate(builder.doc)
+
+          return render plain: errors.map(&:message).join("\n") if errors.any?
+        end
+
+        render xml: builder
+      end
+
+      private
+
+      def sample_oml(xml)
+        xml.lom(xmlns: 'http://ltsc.ieee.org/xsd/LOM') do
+          oml_general(xml)
+          oml_lifecycle(xml)
+          oml_meta_metadata(xml)
+          oml_technical(xml)
+          oml_educational(xml)
+          oml_rights(xml)
+          # Other top-level elements: relation, annotation, and classification.
```

Comment:
@MrSerth I guess we should discuss the used/unused elements in person. Also, I left a couple of TODO notes in this file. Most of them are future work.

Reply:
Fine for me. As discussed:

- *Relation*: We can only add relational information once the task model supports versioning or linking between different tasks. Hence, this is blocked for now.
- *Annotation*: This could be done right now and would be syntactically correct. However, the ratings we intended don't match the semantical XSD.
- *Classification*: The XSD requires the use of a predefined vocabulary. The ProFormA MD 1.0 format uses its own vocabulary which is not valid for strict LOM. In the future, we might use `customElements` instead.

Result: We don't add these top-level elements for now.